md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -894,4 +895,27 @@ RestWrapper insertClone(@PathVariable("id") Integer processId, Principal princip
     public Object execute(String[] params) {
         return null;
     }
+
+
+
+    @RequestMapping(value = {"/SLAMonitoring/{id}", "/SLAMonitoring/{id}/"}, method = RequestMethod.GET)
+    @ResponseBody public
+    RestWrapper SLAMonitoring(@PathVariable("id") Integer processId
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00100)